appendices/package-approval-template.md

```diff
@@ -15,10 +20,21 @@ It looks like you would like to submit this package to JOSS. Here are the next s
 - [ ] Login to the JOSS website and fill out the JOSS submission form using your Zenodo DOI. **When you fill out the form, be sure to mention and link to the approved pyOpenSci review.** JOSS will tag your package for expedited review if it is already pyOpenSci approved.
 - [ ] Wait for a JOSS editor to approve the presubmission (which includes a scope check).
 - [ ] Once the package is approved by JOSS, you will be given instructions by JOSS about updating the citation information in your README file.
-- [ ] When the JOSS review is complete, add a comment to your review in the pyOpenSci software-review repo that it has been approved by JOSS.
+- [ ] When the JOSS review is complete, add a comment to your review in the pyOpenSci software-review repo here that it has been approved by JOSS. An editor will then add the JOSS-approved label to this issue.
```

Comment:
How do you feel about asking the editor to subscribe to the JOSS submission issue? Having two people keeping an eye on the matter could be safer, right?

Reply:
If you think as an editor that it's a reasonable ask, i think it could be really good!! then it would help the editor keep an eye out on the submission. @ocefpaf and i had a brief chat about this in slack - how if JOSS takes a long time it would be easy to end up with stale issues or to forget (if you are me i'd forget :) )

Reply:
ok I will add that now. it's a good idea and not a huge amount of effort to see the emails coming through and keep tabs on the review.